pithos/GstPlayer.py

```diff
@@ -0,0 +1,667 @@
+        blurb='gain for the frequency band 59 Hz',
+        flags=GObject.ParamFlags.READWRITE,
+    )
+
+    eq_band2 = GObject.Property(
+        type=GObject.TYPE_DOUBLE,
+        default=0.0,
+        minimum=-24.0,
+        maximum=12.0,
+        nick='eq-band2 prop',
+        blurb='gain for the frequency band 119 Hz',
+        flags=GObject.ParamFlags.READWRITE,
+    )
+
+    eq_band3 = GObject.Property(
+        type=GObject.TYPE_DOUBLE,
+        default=0.0,
+        minimum=-24.0,
+        maximum=12.0,
+        nick='eq-band3 prop',
+        blurb='gain for the frequency band 237 Hz',
+        flags=GObject.ParamFlags.READWRITE,
+    )
+
+    eq_band4 = GObject.Property(
+        type=GObject.TYPE_DOUBLE,
+        default=0.0,
+        minimum=-24.0,
+        maximum=12.0,
+        nick='eq-band4 prop',
+        blurb='gain for the frequency band 474 Hz',
+        flags=GObject.ParamFlags.READWRITE,
+    )
+
+    eq_band5 = GObject.Property(
+        type=GObject.TYPE_DOUBLE,
+        default=0.0,
+        minimum=-24.0,
+        maximum=12.0,
+        nick='eq-band5 prop',
+        blurb='gain for the frequency band 947 Hz',
+        flags=GObject.ParamFlags.READWRITE,
+    )
+
+    eq_band6 = GObject.Property(
+        type=GObject.TYPE_DOUBLE,
+        default=0.0,
+        minimum=-24.0,
+        maximum=12.0,
+        nick='eq-band6 prop',
+        blurb='gain for the frequency band 1889 Hz',
+        flags=GObject.ParamFlags.READWRITE,
+    )
+
+    eq_band7 = GObject.Property(
+        type=GObject.TYPE_DOUBLE,
+        default=0.0,
+        minimum=-24.0,
+        maximum=12.0,
+        nick='eq-band7 prop',
+        blurb='gain for the frequency band 3770 Hz',
+        flags=GObject.ParamFlags.READWRITE,
+    )
+
+    eq_band8 = GObject.Property(
+        type=GObject.TYPE_DOUBLE,
+        default=0.0,
+        minimum=-24.0,
+        maximum=12.0,
+        nick='eq-band8 prop',
+        blurb='gain for the frequency band 7523 Hz',
+        flags=GObject.ParamFlags.READWRITE,
+    )
+
+    eq_band9 = GObject.Property(
+        type=GObject.TYPE_DOUBLE,
+        default=0.0,
+        minimum=-24.0,
+        maximum=12.0,
+        nick='eq-band9 prop',
+        blurb='gain for the frequency band 15011 Hz',
+        flags=GObject.ParamFlags.READWRITE,
+    )
+
+    volume = GObject.Property(
+        type=GObject.TYPE_DOUBLE,
+        default=1.0,
+        minimum=0.0,
+        maximum=1.0,
+        nick='volume prop',
+        blurb='volume factor',
+        flags=GObject.ParamFlags.READWRITE,
+    )
+
+    __query_duration = Gst.Query.new_duration(Gst.Format.TIME)
+    __query_position = Gst.Query.new_position(Gst.Format.TIME)
+    __query_buffer = Gst.Query.new_buffering(Gst.Format.PERCENT)
+    _desired_state = PseudoGst.STOPPED
+    _actual_state = PseudoGst.STOPPED
+    _prerolled = False
+    _got_duration = False
+    _disk_cached = False
+    _buffering_timer_id = 0
+    _current_bitrate = 0
+    _start_time = 0
```

Comment:
These should be instance variables?

Reply:
They can be, it really doesn't matter there's only ever 1 player?

Reply:
Done. I moved them.